### ISSUE TITLE
extraneous ')' before ';'

### ISSUE THREAD

**Issue by brianyu28** [cs50/help50#138]
```
clang -ggdb3 -O0 -std=c11 -Wall -Werror -Wshadow    credit1.c  -lcs50 -lm -o credit1
credit1.c:144:52: error: extraneous ')' before ';'
        digit = (number % (tracker)) / (tracker/10));
                                                   ^
1 error generated.
make: *** [credit1] Error 1
```

```
clang -ggdb3 -O0 -std=c11 -Wall -Werror -Wshadow    initials.c  -lcs50 -lm -lcrypt -o initials
initials.c:15:27: error: extraneous ')' before ';'
    char c = toupper(s[0]));
                          ^
1 error generated.
make: *** [initials] Error 1
```

```
clang -ggdb3 -O0 -std=c11 -Wall -Werror -Wshadow    vigenere.c  -lcs50 -lm -lcrypt -o vigenere
vigenere.c:16:32: error: extraneous ')' before ';'
        k[i]=(toupper(k[i]))-65);
                               ^
1 error generated.
make: *** [vigenere] Error 1
```

**Comment by mjopp** [comment]
@dmalan 

I think the status of this Issue should be changed from Open to Closed.
You deployed a fix #208 on September 28th, which solves this Issue.

**Comment by dmalan** [comment]
Ah, thanks!
